client/my-sites/plans-v2/utils.ts

```diff
@@ -182,3 +187,9 @@ export function itemToSelectorProduct(
 	}
 	return null;
 }
+
+export function getRealtimeFromDaily(
+	productSlug: JetpackOfferDailyPlans
+): JetpackOfferRealtimePlans | null {
+	return DAILY_PLAN_TO_REALTIME_PLAN[ productSlug ];
```

Comment:
Would love to see this even lay a foundation for other products that have similar upsells, such as Backup. Doesn't have to be inclusive but changing types down the road could lead to some headaches.